taxi/views.py

```diff
@@ -1,3 +1,17 @@
+
+from django.http import HttpRequest
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Car, Manufacturer
+
+
+def index(request: HttpRequest) -> render:
```

Comment:
`render` function actually returns an `HttpResponse` object, so the proper return type annotation is `HttpResponse`, not `render`.